tools/windows-node-installer/go.mod

```diff
@@ -12,16 +12,18 @@ require (
 	github.com/aws/aws-sdk-go v1.23.2
 	github.com/coreos/etcd v3.3.10+incompatible
 	github.com/deckarep/golang-set v1.7.1 // indirect
-	github.com/go-logr/zapr v0.1.0
+	github.com/go-logr/zapr v0.1.1
 	github.com/googleapis/gnostic v0.3.1 // indirect
-	github.com/imdario/mergo v0.3.7 // indirect
 	github.com/openshift/api v3.9.1-0.20190814194116-a94e914914f4+incompatible
 	github.com/openshift/client-go v0.0.0-20190813201236-5a5508328169
+	github.com/operator-framework/operator-marketplace v0.0.0-20190919183128-4ef67b2f50e9 // indirect
+	github.com/operator-framework/operator-sdk v0.10.0 // indirect
```

Comment:
I am a nack on this. They shouldn't be pulled in.

Reply:
they should not. I will drop them manually.